src/App.js

```diff
@@ -0,0 +1,26 @@
+import './styles/App.css';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Router from './routes/routes';
+import { fetchCarReservations, fetchCars } from './redux/thunk';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this functional component to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

That way, your code will meet the latest standards. 💪